Sources/_OpenAPIGeneratorCore/Translator/Responses/translateResponse.swift

```diff
@@ -103,6 +103,61 @@ extension TypesFileTranslator {
                     )
                 )
                 bodyCases.append(bodyCase)
+
+                var throwingGetterSwitchCases = [
+                    SwitchCaseDescription(
+                        kind: .case(.identifier(".\(identifier)"), ["body"]),
+                        body: [.expression(.return(.identifier("body")))]
+                    )
+                ]
+                // We only generate the default branch if there is more than one case to prevent
+                // a warning when compiling the generated code.
+                if typedContents.count > 1 {
+                    throwingGetterSwitchCases.append(
+                        SwitchCaseDescription(
+                            kind: .default,
+                            body: [
+                                .expression(
+                                    .try(
+                                        .identifier("throwUnexpectedResponseBody")
+                                            .call([
+                                                .init(
+                                                    label: "expectedContent",
+                                                    expression: .literal(.string(contentType.headerValueForSending))
+                                                ),
+                                                .init(label: "body", expression: .identifier("self")),
+                                            ])
+                                    )
+                                )
+                            ]
+                        )
+                    )
+                }
+                let throwingGetter = VariableDescription(
+                    accessModifier: config.access,
+                    isStatic: false,
+                    kind: .var,
+                    left: identifier,
+                    type: associatedType.fullyQualifiedSwiftName,
+                    getter: [
+                        .expression(
+                            .switch(
+                                switchedExpression: .identifier("self"),
+                                cases: throwingGetterSwitchCases
+                            )
+                        )
+                    ],
+                    getterEffects: [.throws]
+                )
+                let throwingGetterComment = Comment.doc(
+                    """
+                    The associated value of the enum case if `self` is `.\(identifier)`.
+
+                    - Throws: Runtime error if `self` is not `.\(identifier)`.
```

Comment:
```suggestion
                    - Throws: An error if `self` is not `.\(identifier)`.
```

Since "runtime error" isn't a public type, I wouldn't mention it here.

Reply:
OK, happy to change. FTR I didn't mean `RuntimeError`, I meant "a runtime error" (in the generic sense), but I guess the capital R led to confusion. Will update everywhere (manually, because it affects the reference code).

Resolved in 4c423d8.